osu.Game/Overlays/Rankings/Tables/RankingsTable.cs

```diff
@@ -47,10 +48,11 @@ private void load()
             {
                 RelativeSizeAxes = Axes.Both,
                 Depth = 1f,
-                Margin = new MarginPadding { Top = row_height }
+                Margin = new MarginPadding { Top = row_height + row_spacing },
+                Spacing = new Vector2(0, row_spacing),
             });
 
-            rankings.ForEach(_ => backgroundFlow.Add(new TableRowBackground()));
+            rankings.ForEach(_ => backgroundFlow.Add(new TableRowBackground(row_height)));
```

Comment:
Not much point passing this in the ctor if you have already-public properties to work with:
```suggestion
            rankings.ForEach(_ => backgroundFlow.Add(new TableRowBackground { Height = row_height }));
```